### PR TITLE
[DOC] Related for Array#[]

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: array.c [hunk 1]
```diff
@@ -8567,7 +8568,7 @@ rb_ary_deconstruct(VALUE ary)
  *
  *  These methods do not modify +self+.
  *
- *  - #[]: Returns one or more elements.
+ *  - #[] (aliased as #slice): Returns consecutive elements as determined by a given argument.
```
Comment (peterzhu2118) [inline array.c:8571]: The other aliases are displayed as a list (e.g. `#select` and `#filter`), but this one is different. I don't have a strong opinion on how we display aliases, but can we make it consistent?

Comment (BurdetteLamar) [reply array.c:8571]: I like this way better.  Will add something in the doc guide.

If I try to resolve these here, I will almost certainly foul my repo and branches.  Can we merge this and let me fix afterwards?

Comment (peterzhu2118) [reply array.c:8571]: Ok, but can we be consistent in this PR and you can change it back in a future PR?

Comment (BurdetteLamar) [reply array.c]: Fixed.

**Pull request merged**